src/test/java/touro/chess/BishopPieceTest.java

```diff
@@ -0,0 +1,109 @@
+package touro.chess;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class BishopPieceTest
+{
+    @Test
+    public void getMoves_upperLeft()
```

Comment:
You don't need 4 different tests for the Location (4,4), you should do this all in one test.